pagination/keysetpagination/paginator.go

```diff
@@ -62,7 +62,8 @@ func Paginate[I Item](p *Paginator) pop.ScopeFunc {
 	return func(q *pop.Query) *pop.Query {
 		return q.
 			Limit(p.Size()+1).
-			Where(fmt.Sprintf("%s > ?", id), p.Token())
+			Where(fmt.Sprintf(`%q > ?`, id), p.Token()).
+			Order(fmt.Sprintf(`%q ASC`, id))
```

Comment:
This should be:

```
			Where(fmt.Sprintf("`%s` > ?", id), p.Token()).
			Order(fmt.Sprintf("`%s` ASC", id))
```

Reply:
If it doesn't work across databases we will have to add a switch statement for the dialect! I think pop also has an escape helper if I'm not mistaken.